### PR TITLE
Use MHCgnomes for allele parsing instead of MHCnames

### PR DESCRIPTION
**Pull request by iskandr**
* Created a `common.normalize_allele_name` helper function which uses `mhcgnomes` instead of `mhcnames`. 
* Updated the baseline 34mer pseudosequences to use NetMHCpan 4.1's sequences

### CODE REVIEW

File: downloads-generation/allele_sequences/filter_sequences.py [hunk 1]
```diff
@@ -64,28 +49,46 @@ def run():
...
-                continue
-            if normalized in seen:
-                continue
-            if any(n in name for n in class_ii_names):
-                print("Dropping", name)
+                else:
+                    print("Unable to parse: %s" % name)
+                    continue
+            else:
+                if type(result) not in (Allele, AlleleWithoutGene, Gene):
```
Comment (timodonnell) [inline downloads-generation/allele_sequences/filter_sequences.py:64]: curious if you have an example for a `Gene` result? Should we really be including those or just skipping those sequences?

Comment (iskandr) [reply downloads-generation/allele_sequences/filter_sequences.py]: Invariant genes sometimes don't have alleles. The one that comes to mind is "BoLA-DRA" but I've also seen it for non-canonical murine genes.

**Review by timodonnell — COMMENTED** [review]
Looks good to me, left some small comments and added a commit to renormalize allele sequence names when loading the predictor as discussed. If mhcgnomes can be sped up to normalize all the allele names faster (currently 8 sec) without too much work, that would be great. Thanks for going through this slog @iskandr !

**Comment by timodonnell** [comment]
Also just to confirm I think we can merge this without re-training models, to avoid having this PR go stale if it takes a while to get to that

File: mhcflurry/common.py [hunk 2]
```diff
@@ -7,10 +7,50 @@
 
 import numpy
 import pandas
+from mhcgnomes import parse, Allele, AlleleWithoutGene, Gene, Class2Pair
 
 from . import amino_acid
 
 
+def normalize_allele_name(
```
[the remark this answers is not in the document]
Comment (iskandr) [reply mhcflurry/common.py:15]: 👍

File: downloads-generation/allele_sequences/GENERATE.sh [hunk 3]
```diff
@@ -43,7 +45,7 @@ cp $SCRIPT_ABSOLUTE_PATH .
 # pseudosequences but have associated training data
 TRAINING_DATA="$(mhcflurry-downloads path data_curated)/curated_training_data.csv.bz2"
 
-python select_alleles_to_disambiguate.py \
+python3 select_alleles_to_disambiguate.py \
```
[the remark this answers is not in the document]
Comment (iskandr) [reply downloads-generation/allele_sequences/GENERATE.sh]: 👍

**Comment by iskandr** [comment]
> Also just to confirm I think we can merge this without re-training models, to avoid having this PR go stale if it takes a while to get to that

Sounds good to me.

**Comment by timodonnell** [comment]
@iskandr good to merge?

**Comment by iskandr** [comment]
Yes!

**Comment by timodonnell** [comment]
Nice to have this in - thanks @iskandr !

**Pull request merged**